FeedStoreChallenge/Infrastructure/ManagedCache.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Fernando Goulart on 01/08/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
```

Comment:
`internal` is a default access level, no need to declare it explicitly. 
Also this class is not intended to be inherited thus it should be `final`

Please remove `internal` modifier and make the class `final`.

```suggestion
final class ManagedCache: NSManagedObject {
```

Reply:
Done. Thanks.